cmd/build.sh

```diff
@@ -125,7 +125,7 @@ do
 
     echo -en "\t - $goos/$goarch..."
 
-    zipfile="$filename-$build_version-$pos-$parch"
+    zipfile="wso2$filename-cli-$build_version-$pos-$parch"
```

Comment:
@menuka94  What does variable **filename** refers to here ? Can't we change that itself ?

This will change only the external zip name and after extracting we will have folder **mi** and **mi.exe** inside. Aren't we changing that too as **wso2mi-cli**.
@a5anka Thoughts please ?

Reply:
> @menuka94 What does variable **filename** refers to here ? Can't we change that itself ?

No. `$filename` refers to the main `.go` file (in this project it's `mi.go`. If we change that to `wso2mi-cli.go`, then the CLI itself becomes `wso2mi-cli <command>`.

> This will change only the external zip name and after extracting we will have folder **mi** and **mi.exe** inside. Aren't we changing that too as **wso2mi-cli**.

That could be done if needed.

Reply:
@Nirothipan the purpose was to just change the archive name to avoid confusion.